config/enabled.yml

```diff
@@ -1408,6 +1408,10 @@ Performance/RedundantSortBy:
   Description: 'Use `sort` instead of `sort_by { |x| x }`.'
   Enabled: true
 
+Performance/RegexpMatch:
+  Description: 'Use `match?` instead of `Regexp#match`, `String#match` or `=~` when `MatchData` is not used.'
```

Comment:
We might be better off not explicitly listing `Regexp#match` and `String#match` since `match?` is left more generic. This does not have reference to `Symbol#match`. Alternatively, we might want to be more explicit about saying use `Regexp#match?`, `String#match?`, or `Symbol#match?`.